src/dbnode/storage/database.go

```diff
@@ -232,10 +232,10 @@ func (d *db) updateSchemaRegistry(newNamespaces namespace.Map) error {
 	schemaUpdates := newNamespaces.Metadatas()
 	merr := xerrors.NewMultiError()
 	for _, metadata := range schemaUpdates {
-		curSchemaId := "none"
+		curSchemaID := "none"
 		curSchema, err := schemaReg.GetLatestSchema(metadata.ID())
 		if curSchema != nil {
-			curSchemaId = curSchema.DeployId()
+			curSchemaID = curSchema.DeployId()
```

Comment:
not from your changes but looks like the error on line 236 is unchecked. what's the deal there?

Reply:
Hm, not immediately obvious to me what to do on error, opened #1694.